app/src/main/java/com/example/byitscover/scrapers/GoogleScraper.java

```diff
@@ -50,12 +52,41 @@ public GoogleScraper() {
         searchingUrl.replaceAll("[\\n]", "");
         Document document = Jsoup.connect(searchingUrl).get();
 
-        //go to first search result link
-        Element link = (Element) document.select("div.g").first().childNode(1)
-                .childNode(0).childNode(0);
-        String bookUrl = link.attr("abs:href");
+        Elements links = (Elements) document.select("div.g");
+
+        List<BookListing> listings = new ArrayList<BookListing>();
+        BookListing firstListing = getListingFromElement(links, 0, query);
+        listings.add(firstListing);
+
+        if (links.size() > 1) {
+            BookListing secondListing = getListingFromElement(links, 1, query);
+            listings.add(secondListing);
+        }
+        if (links.size() > 2) {
+            BookListing thirdListing = getListingFromElement(links, 2, query);
+            listings.add(thirdListing);
+        }
+
+        return listings;
+    }
+
+    private BookListing getListingFromElement(Elements links, int i, Query query) throws MalformedURLException {
+        Element linkElement;
+        if (i == 0) {
+            linkElement = (Element) links.get(i).childNode(1).childNode(0).childNode(0);
+        }
+        else {
+            linkElement = (Element) links.get(i).childNode(0).childNode(0).childNode(0);
+        }
+
+        String bookUrl = linkElement.attr("abs:href");
         System.out.println(bookUrl);
```

Comment:
Another print that might need to be removed

Reply:
Same as above